lib/util.js

```diff
@@ -477,7 +477,7 @@ function stylizeWithColor(str, styleType) {
   return str;
 }
 
-function stylizeNoColor(str, styleType) {
+function stylizeNoColor(str) {
```

Comment:
I believe this is done for performance/efficiency as this method (used as the default for the `stylize` config option) is called with two arguments and AFAIK V8 still prefers when the number of parameters and and number of arguments match and/or are consistent.